doc/rtd/topics/bugs.rst

```diff
@@ -0,0 +1,95 @@
+.. _reporting_bugs:
+
+Reporting Bugs
+**************
+
+Logs
+====
+
+To aid in debugging collecting the necessary logs, first run the
+`collect-logs` subcommand to produce a tarfile that you can easily upload
+to aid in intial debugging of your issue:
```

Comment:
```suggestion
to aid in initial debugging of your issue:
```

Reply:
This is a typo, but we also use "aid in [initial] debugging" twice in this sentence, which isn't likely to... aid in the reader's understanding. :grin:

Reply:
Cleaned up the entire paragraph to avoid duplication